blog/2023-06-14-paralus-at-platformcon-2023/index.md

```diff
@@ -0,0 +1,114 @@
+
+A bit about “Platform engineering” is a discipline focused on building and maintaining the underlying infrastructure and tools that enable developers to build, deploy, and run software applications efficiently. It involves creating a stable and scalable platform that provides a foundation for software development, deployment, and operations.
+
+**[Six Guiding Principles For Platform Teams](https://www.forbes.com/sites/forbestechcouncil/2023/02/23/how-to-empower-modern-kubernetes-management-with-a-platform-team-model/?sh=1690484c52c7)**: 
+- Consistent Automation
+- Security Preparedness
+- Centralized Visibility
+- Governance
+- Kubernetes Distribution Flexibility
+- Oriented To Self-Service
+
+*In this blog we will be referring to more on **Kubernetes access management** or better to call it **Security Preparedness:** *
+
+With the right access to the right roles, separation of duties becomes easier to implement and enforce, and it becomes easier for the platform and security teams to protect against unauthorized use.  
+
+## Why is to Secure Access to k8s Important? 
+
+The inability to secure K8s infrastructure is a growing problem for organizations. In May 2022, a non-profit security organization named [The Shadowserver Foundation](https://www.shadowserver.org/news/over-380-000-open-kubernetes-api-servers/) scanned more than 450,000 systems hosting K8s and found more than 380,000 (84 percent) of these systems were accessible via the Internet, potentially providing an opening into a corporate network. In fact, the data shows that the majority of K8s API servers are found in the United States (nearly 53 percent).
+
+
+## Traditional challenges of Kubernetes access management
+
+Access management in Kubernetes can pose several challenges, some of which are considered traditional pain points for administrators:
+
+**Complexity**: Kubernetes has a complex access control model, involving multiple components and concepts like Roles, RoleBindings, ServiceAccounts, and ClusterRoles. Understanding and managing these components can be challenging, especially for beginners or organizations without prior experience with Kubernetes.
+
+**Granularity vs. Complexity Tradeoff**: Kubernetes provides fine-grained access control, allowing administrators to define highly specific access policies. However, this level of granularity can lead to increased complexity in managing and maintaining access control rules. Balancing the need for fine-grained access control with simplicity and maintainability is a common challenge.
+
+**Role-Based Access Control (RBAC) Configuration**: Configuring RBAC policies accurately and efficiently is crucial for secure access management. Designing and implementing RBAC rules that align with an organization's security requirements can be complex, particularly when dealing with a large number of users, teams, and applications.
```

Comment:
This point and the earlier point on `Granularity vs Complexity Tradeoff` is similar, let us remove one of them or talk something else.

Reply:
Same here, let us add these as bullet points so that it's easy to read and comprehend.